package.json

```diff
@@ -93,6 +93,7 @@
     "@styled-system/theme-get": "5.1.2",
     "@swc-node/register": "^1.10.9",
     "@swc/helpers": "^0.5.15",
+    "@unleash/proxy-client-react": "^4.5.2",
```

Comment:
In case it saves others some confusion, I want to clarify a non-obvious point. While the [unleash proxy](https://docs.getunleash.io/reference/unleash-proxy) is deprecated, the `proxy-client`s are not. They are still necessary to interface with our self-hosted `unleash` instances. So we can now spin down the `artsy-unleash-web-proxy` [deployment](https://github.com/artsy/artsy-unleash/blob/main/hokusai/production.yml#L165) and replace it with `unleash edge` or just point request directly to our [web deployment](https://github.com/artsy/artsy-unleash/blob/main/hokusai/production.yml#L4)

Reply:
> replace it with unleash edge or just point requests directly to our [web deployment](https://github.com/artsy/artsy-unleash/blob/main/hokusai/production.yml#L4)

I'm confused about this. I thought Edge replaced the proxy. Are you saying Edge is unnecessary and the non-proxy service can also handle requests from clients?

Reply:
> Are you saying Edge is unnecessary and the non-proxy service can also handle requests from clients?

@joeyAghion, yeah, thats correct. They have a bit of a breakdown [here](https://docs.getunleash.io/reference/front-end-api). The main difference seems to be scalability. If standing up an edge instance up proves straight forward (and I think it will), it may make sense to just do it as part of this work, but for testing and getting agreement, I just started with the frontend api.

Reply:
Also, given we were hitting it for every request previously, I would think it should be handle the lower throughput.

Reply:
Was that actually the case? My understanding was that calls to the client depended only on the already-loaded toggle state, and that state would be refreshed from the server asynchronously based on the configured `refreshInterval`.